Example/Emission/UnroutedViewController.m

```diff
@@ -27,7 +27,7 @@ - (instancetype)initWithRoute:(NSString *)route
 
 - (NSURL *)urlForRoute:(NSString *)route
 {
-  if ([route containsString:@"https"]) {
+  if ([route containsString:@"http"]) {
```

Comment:
@alloy here's that Example app fix.

Reply:
I think that for the example app we can just do both, but maybe in the case of `http` log a warning about it? Use `NSLog` for that.